Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XldbDataStore.cs

```diff
@@ -56,9 +54,9 @@ public class XLGppDataStore: IDisposable
         /// Gets all the events of a certain type from the DB
         /// </summary>
         /// <returns>List of event objects recovered from DB </returns>
-        public IEnumerable<string> GetEventsByType(int eventTypeID)
+        public IEnumerable<string> GetEventsByType(Execution.Analyzer.Xldb.ExecutionEventId eventTypeID)

```

Comment:
>Execution.Analyzer.Xldb [](start = 51, length = 23)

pick one - full name here or `using`

Reply:
Well, its a different namespace alltogether ("namespace BuildXL.Analyzers.Core.XLGPlusPlus"), which is why it needed the full name here. 

---
In reply to: [309369609](https://github.com/microsoft/BuildXL/pull/671#discussion_r309369609) [](ancestors = 309369609)